sdcm/ec2_client.py

```diff
@@ -61,7 +60,8 @@ def _get_ec2_client(self, region_name=None):
             boto3.setup_default_session(region_name=region_name)
             return self._get_ec2_client()
 
-    def _request_spot_instance(self, instance_type, image_id, region_name, network_if, spot_price, key_pair='',  # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments
```

Comment:
I think move it like this, would effect the rest of the function in this class

Reply:
This change not affected on function code block?